src/components/layer-controller/LayerControllerSubscriber.js

```diff
@@ -102,9 +102,10 @@ function LayerControllerSubscriber(props) {
     setRasterLayers(newLayers);
   }
 
-  const shouldWaitForRaster = loaders[dataset].loaders.raster;
+  const hasBitmask = (imageLayerMeta.length ? imageLayerMeta : [{ metadata: { isBitmask: true } }])
+    .every(l => !l?.metadata?.isBitmask);
   // Only want to show vector cells controller if there is no bitmask
-  const canShowCellVecmask = shouldWaitForRaster ? (rasterLayers || [{ type: 'bitmask' }]).findIndex(l => l.type === 'bitmask') < 0 : true;
+  const canShowCellVecmask = hasBitmask;
```

Comment:
```suggestion
  const canShowCellVecmask = !hasBitmask;
```

Based on the comment above (line 107), should this be `!hasBitmask`?